scripts/examplegen/main.go

```diff
@@ -92,11 +92,6 @@ func run(lint bool) error {
 		w = io.Discard
 	}
 
-	_, err = fmt.Fprint(w, "// Code generated by examplegen. DO NOT EDIT.\n")
-	if err != nil {
-		return err
-	}
```

Comment:
why are we removing this?

Reply:
Because the comment made the `markdownlint` cry its not a valid YAML. YAML can't have comments.

Reply:
We can ignore this file altogether as its a generated file.

Reply:
yaml can absolutely have comments. that's an insane thing for it to insist. 😅

Reply:
Isn't this is JSON and not YAML?

Reply:
Oops. Thanks for correcting Cian. It was an oversight from me. For some reason `prettier` was happy with comments in JSON but `biome` is not.

I am going to remove the comment.